tests/unit/test_charm.py

```diff
@@ -58,8 +58,39 @@ def test_on_config_changes_aborts_if_host_is_container(harness, mocker):
     assert not harness.charm._stored.started
 
 
+def test_get_initiator_name_from_file(harness, mocker, tmp_path):
```

Comment:
In this unit test. I expect to test
1. happy case:  different content of the file, we should get different initiator name
2. error case:  ex: file is not existed, file is empty,  format is wrong  => return None

We can add more error cases.

Reply:
The test currently has the cases for 
1. File isn't present, return None
2. File has content properly, return initiator name.

I'll add these extra error cases where the method should return `None`.
* File is empty
* File has wrong format, eg: instead of `initiatorName` there's a wrong key in the file.